sdks/java/core/src/test/java/org/apache/beam/sdk/util/IOChannelUtilsTest.java

```diff
@@ -80,6 +80,13 @@ public void testLargeShardCount() {
   }
 
   @Test
+  public void testFilePrefix() throws Exception {
+    IOChannelUtils.getFactory("file://tmp");
+    IOChannelUtils.getFactory("file:/tmp");
+    IOChannelUtils.getFactory("file:tmp");
```

Comment:
Thanks guys. Let me add some deeper tests. Regarding @tgroh comment, does it mean that I have to implement some parsing on the path to deal with `$PWD/file:/foo/bar`?